content/manuals/compose/install/_index.md

```diff
@@ -30,6 +30,17 @@ Docker Desktop is available on:
 
 If you have already installed Docker Desktop, you can check which version of Compose you have by selecting **About Docker Desktop** from the Docker menu {{< inline-image src="../../desktop/images/whale-x.svg" alt="whale menu" >}}.
 
+> [!NOTE]
+>
+> After removal of End-of-Lived Docker Compose V1 in [4.23.0](/desktop/release-notes/#4230), `docker-compose` command points directly to Docker Compose V2 binary, ran in standalone mode. 
+> If you rely on Docker Desktop auto-update, the symlink might be broken and command unavailable, as update doesn't ask for administrator password. 
+> To fix it, just recreate the symlink:
+> ```console
+> $ sudo rm /usr/local/bin/docker-compose
+> $ sudo ln -s /Applications/Docker.app/Contents/Resources/cli-plugins/docker-compose /usr/local/bin/docker-compose
+> ```
```

Comment:
For users who don't have admin right on their machine we should mention the alias creation as an another workaround, no?
Maybe we should also specify this only affect Mac users

Reply:
IMHO better not give them a non-admin workaround, as installation would then diverge from the canonical one, and let their super-admin manage the required update in its company-wide orchestration automation

Reply:
Right, make sense 👌